modules/oadp-operator-supported.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-intro.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="oadp-operator-supported_{context}"]
+= Support for {oadp-full}
+
+.Supported versions of {oadp-short}
+[width="100%",cols="10%,12%,12%,13%,13%,20%,20%, options="header",]
+|===
+
+|Version
+|{OCP-short} version
+|General availability
+|Full support ends
+|Maintenance ends
+|Extended Update (EUS)
+|Extended Update Support Term 2 (EUS Term 2)
+
+
+
+|1.4
+a|
+* 4.14
+* 4.15
+* 4.16
+|10 Jul 2024
+|Release of 1.5
+|Release of 1.6
+a|
+27 Jun 2026
+
+EU must be on {OCP-short} 4.16
```

Comment:
?
```suggestion
EUS must be on {OCP-short} 4.16
```
I'm not sure if there's a real difference between EU vs EUS, I've only ever seen it as "EUS", so I'm assuming this is just a typo.